ReactWindows/ReactNative.Shared/DevSupport/DevSupportManager.cs

```diff
@@ -346,21 +346,36 @@ public Task<bool> IsPackagerRunningAsync()
             return _devServerHelper.IsPackagerRunningAsync();
         }
 
+#if WINDOWS_UWP
+        static ProgressDialog _currentprogressDialog = null; // UWP can only have one
+#endif
+
         public async void HandleReloadJavaScript()
         {
             DispatcherHelpers.AssertOnDispatcher();
 
             HideRedboxDialog();
-            HideDevOptionsDialog();
 
```

Comment:
That must be a merge issue? I did not intend to remove that at all. I will check into what happened there. Thanks.